### ISSUE TITLE
[bug][ui]: Labels color selector missing keyboard focus utils

### ISSUE THREAD

**Issue by rickarize** [Arize-ai/phoenix#10275]
<img width="521" height="453" alt="Image" src="https://github.com/user-attachments/assets/6667a2bb-97a4-42f0-a018-6e3de9fafb29" />

expected behavior: hitting tab from description would highlight color group, which should be a focus trap. hitting enter would let you enter the group, tab or arrows to move through squares, enter to select a square, esc to leave focus trap.

Tony mentions that this could be a radio group that happens to have a swatch styling layer on top, which makes sense.

**Comment by vickywane** [comment]
@rickarize the styling for the `<ColorSwatchPicker />` component has it's outline disabled. The outline can natively handle focus trap for the listed color swatches. I removed the outline in the stylesheet, and this works as expected.  

@mikeldking @cephalization 

https://github.com/user-attachments/assets/d9f6860e-a96b-466b-b5fa-229f76cc0749

**Comment by cephalization** [comment]
@vickywane feel free to post a PR, thanks for looking into this

**Comment by vickywane** [comment]
@cephalization I've raised a PR.

**Comment by cephalization** [comment]
@vickywane merged! Thanks for your contribution
